07-documenting_R.Rmd

```diff
@@ -14,7 +14,9 @@ This chapter is aimed at authors and potential authors of R documentation provid
 
 ## Guidelines for writing R help files
 
-Following are the suggested guidelines while creating the system help files in the .Rd format. A example of the .Rd format is available for [mean.Rd](https://svn.r-project.org/R/trunk/src/library/base/man/mean.Rd). These are intended for writing the core documentations but may also be useful for package writers. Extensive details of writing R documentation files can be found in the [Writing R Extensions](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Writing-R-documentation-files) manual.
+This section summarises the [recommended style for writing the R help files](https://developer.r-project.org/Rds.html). The R help files are written in “R documentation” (Rd) format, a markup language which resembles LaTeX. An example of the .Rd format is available for [mean.Rd](https://svn.r-project.org/R/trunk/src/library/base/man/mean.Rd). These are intended for writing the core documentations but may also be useful for package writers. Extensive details of writing R documentation files can be found in the [Writing R Extensions](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Writing-R-documentation-files) manual.
+
+Note: Many R package authors use the R package [roxygen](https://cran.r-project.org/web/packages/roxygen2/vignettes/roxygen2.html) to write documentation (which eventually creates the .Rd files), however, in this guide we focus on .Rd files only because the manual pages for the base distribution are written directly as .Rd files.
 
```

Comment:
I would suggest to replace

R package [roxygen](https://cran.r-project.org/web/packages/roxygen2/vignettes/roxygen2.html)

with 

R package [roxygen2](https://cran.r-project.org/package=roxygen2)

Reply:
Thanks for the review @bettinagruen! I've made the changes and I'm merging this PR now. Hope that it okay! Please feel free to open a new issue if you want to suggest anything else.